README.md

```diff
@@ -2,40 +2,77 @@
 
 The CSS source files also contain [Topdoc](https://github.com/Topdoc/topdoc) comments with a placeholder for documentation values that are injected at build time. The source of those injected values is found in the YAML formatted files in the `docs` folder. A key part of the docs data is the `markup` node, which contains the HTML elements needed to apply the corresponding element selectors and render the elements as generated Topdoc output.
 
 A successful build will create a `dist` folder. The `dist/docs` folder is where the Topdoc output and related template files will end up.
 
-### Updating internal dependencies
+## Using Spectrum CSS
 
-*Internal-to-Adobe only - for now*
+Spectrum CSS can be consumed as whole or in part with three distinct methods of applying colorstops.
 
-Locally clone the `spectrum-css-deps` repo. This repo contains internal, non open source dependencies needed by Spectrum CSS. Keep `spectrum-css-deps` as a sibling repo to `spectrum-css`. In `spectrum-css-deps`, run `npm run build` to generate the dependencies and `npm run copy` to copy the generated dependencies into `spectrum-css`. Then go ahead and review and commit any changes.
+To install Spectrum CSS, run `npm install @adobe/spectrum-css` from your project's root directory. You will be able to find everything you need for both strategies in the `node_modules/@adobe/spectrum-css/` folder.
 
-## Using Spectrum CSS
+### CSS Custom Properties Strategy
+
+Spectrum CSS includes build output that uses CSS custom properties to change themes and scales. This is the preferred approach due to its simplicity and smaller file size, but it does not work in IE 11.
+
+Start by including the base set of variables:
+
+```html
+<!-- Include global variables first -->
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-global.css">
+
+<!-- Include only the scales and colorstops your application needs, be sure to use the *-unique.css files -->
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-medium-unique.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-large-unique.css">
+
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-light-unique.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-dark-unique.css">
+
+<!-- Include index-vars.css for all components you need -->
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/components/icon/index-vars.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.10.0/dist/components/button/index-vars.css">
```

Comment:
Had reservations about including `unpkg` in here, but honestly it makes the usage really obvious and it's copy-paste ready, so I think it's good. Thoughts?